### ISSUE TITLE
Ernie+dnn训练不收敛

### ISSUE THREAD

**Issue by lao-xu** [PaddlePaddle/PaddleNLP#786]
在ErnieModel后面接dnn，但是训练不收敛，或者说看起来没有在训练，不知道哪里错了？
class NLPDNNModel(nn.Layer):
    def __init__(self,  num_classes,  model):
        super().__init__()
        self.model = model
        self.fc1 = nn.Linear(768, num_classes)
    def forward(self, x):
        sequence_output，pool_output = self.model(x)
        y = self.fc1(pool_output)
        return y
model = NLPDNNModel(num_classes=30, model=ErnieModel.from_pretrained(pretrained_model_name_or_path="ernie-2.0-en"))
for epoch in range(1, epochs + 1):
    for step, batch in enumerate(train_data_loader, start=1):
        input_ids,  labels = batch
        pred= model(input_ids)
        loss = F.softmax_with_cross_entropy(pred, labels)
        probs = F.softmax(pred, axis=1)
        correct = metric.compute(probs, labels)
        metric.update(correct)
        acc = metric.accumulate()
        loss.backward()
        optimizer.step()
        optimizer.clear_grad()
        global_step += 1
        if global_step % 10 == 0 :
            print("global step %d, epoch: %d, batch: %d, loss: %.5f, acc: %.5f" % (global_step, epoch, step, loss, acc))
结果：
global step 10, epoch: 1, batch: 10, loss: 4.06760, acc: 0.02876
global step 20, epoch: 1, batch: 20, loss: 3.95304, acc: 0.02869
global step 30, epoch: 1, batch: 30, loss: 3.73309, acc: 0.02884
global step 40, epoch: 1, batch: 40, loss: 3.71727, acc: 0.02893
global step 50, epoch: 1, batch: 50, loss: 3.86115, acc: 0.02867
global step 60, epoch: 1, batch: 60, loss: 3.58650, acc: 0.02876
global step 70, epoch: 1, batch: 70, loss: 3.62628, acc: 0.02865
global step 80, epoch: 1, batch: 80, loss: 3.93945, acc: 0.02864

**Comment by wawltor** [comment]
目前的learning_rate多少了

**Comment by lao-xu** [comment]
> 目前的learning_rate多少了

0.0001， 0.001都试过

**Comment by wawltor** [comment]
> > 目前的learning_rate多少了
> 
> 0.0001， 0.001都试过

如果方便的话 可以提供一份数据给我们 我们来调试看看 因为不同的数据表现不一样

**Comment by lao-xu** [comment]
> > > 目前的learning_rate多少了
> > 
> > 
> > 0.0001， 0.001都试过
> 
> 如果方便的话 可以提供一份数据给我们 我们来调试看看 因为不同的数据表现不一样

好的，请问如何给你们

**Comment by joey12300** [comment]
@lao-xu 您好，可以放在百度云盘或者发到我的个人邮箱136876878@qq.com

**Comment by lao-xu** [comment]
> @lao-xu 您好，可以放在百度云盘或者发到我的个人邮箱[136876878@qq.com](mailto:136876878@qq.com)

已发，谢谢

**Comment by lao-xu** [comment]
> @lao-xu 您好，可以放在百度云盘或者发到我的个人邮箱[136876878@qq.com](mailto:136876878@qq.com)

已解决，谢谢
